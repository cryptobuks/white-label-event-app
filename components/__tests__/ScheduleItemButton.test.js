import React from 'react';
import renderer from 'react-test-renderer';
import ScheduleItemButton from '../ScheduleItem/ScheduleItemButton';

describe('ScheduleItemButton', () => {
  it('renders correctly', () => {
    const comp = <ScheduleItemButton />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
