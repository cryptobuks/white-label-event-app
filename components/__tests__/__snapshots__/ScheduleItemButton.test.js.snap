// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleItemButton renders correctly 1`] = `
<Image
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  source={1}
  style={
    Object {
      "height": 20,
      "width": 20,
    }
  }
  testID={undefined}
/>
`;
